app/pages/project/instances/actions.tsx

```diff
@@ -132,6 +138,13 @@ export const useMakeInstanceActions = (
         },
       ]
     },
-    [projectSelector, deleteInstance, navigate, rebootInstance, startInstance, stopInstance]
+    [
+      project,
+      navigate,
+      deleteInstanceAsync,
+      rebootInstance,
+      startInstance,
+      stopInstanceAsync,
+    ]
```

Comment:
The changes in this file are no longer strictly needed for this PR because I've taken the polling on the list page back out, but they are still very good to do.